src/command/check-secret-expiry.ts

```diff
@@ -3,29 +3,47 @@ import chalk from 'chalk';
 
 import { AWSCredentials, ProfileConfiguration } from '../mfa-login';
 
-import { DEFAULT_SECRET_KEY_EXPIRY_DAYS, getDaysAfterCreated } from '../lib/time';
+import { getKeyAgeInDays } from '../lib/time';
 import { createIAMClient, createStsClient } from '../lib/create-aws-client';
 
 const getExpiryStatus = (
   dateCreated: Date,
-  expiryInDays: number
-): [string | undefined, chalk.Chalk | undefined] => {
-  const days = getDaysAfterCreated(dateCreated.getTime());
-
-  let expiryText;
-  let chalkColor;
-
-  const daysLeft = expiryInDays - days;
+  maxAge: number | undefined
+): { text: string; textColor: chalk.Chalk } | undefined => {
+  let textColor;
+  const keyAge = getKeyAgeInDays(dateCreated.getTime());
+
+  if (maxAge) {
+    const daysLeft = maxAge - keyAge;
+
+    if (daysLeft < 0) {
+      return { text: `✘ has expired`, textColor: chalk.red };
+    } else if ([0, 1].includes(daysLeft)) {
+      return {
+        text: `${daysLeft === 1 ? '⚠ expires tomorrow' : '⚠ expires today'}`,
+        textColor: chalk.yellow,
+      };
+    } else if (daysLeft < 7) {
+      return { text: `⚠ expires in ${daysLeft} days`, textColor: chalk.yellow };
+    } else if (daysLeft < 30) {
+      return {
+        text: `✓ expires in ${keyAge} days`,
+        textColor: chalk.green,
+      };
```

Comment:
we should change all of this text to be clearer that we are approaching key-max-age, and not that the key is actually expiring. Something like Key will reach maximum age in X days maybe? I think that we just don't want to tell the user their keys expired if it actually isn't

Reply:
I agree we should not be talking about expiration. It's also a bit hard to follow all the different paths in this code. Can you provide some examples of the different outputs? Either just a text version in a comment or a screenshot?